plugins/nid-team/rebase/rebase_manager.py

```
@@ -0,0 +1,702 @@
+                    pr_merged = True
+            except Exception:
+                pass
+        
+        # Or check if openshift/master already has a commit matching "Rebase to ..."
+        if not pr_merged:
+            try:
+                log = self.run_cmd(["git", "log", "-n", "10", "--pretty=format:%s"])
+                if "Rebase to " in log:
+                    pr_merged = True
+            except Exception:
+                pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)